VeroliBGV/BGV003.xml

```diff
@@ -100,327 +159,598 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                 is similar to the one preserved in <ref type="mss"
+                                    corresp="ESddm018#a2"/>. 
+                              </desc>
+                              <q xml:lang="gez"> ጸሎተ፡ አርዌ፡ ምድር፡ በስምከ፡ ኪንድ፡ <gap reason="illegible"
+                                 unit="chars" quantity="1"/>ድር፡ ንቋዕ፡ ላላደ፡ ይ ቤፌ<gap
+                                    reason="illegible" unit="chars" quantity="1"/>፡ ጡጡራን፡ ቀረዓዱን፡ ደመ፡
+                                 <gap reason="illegible" unit="chars" quantity="1"/>ራን፡ በከመ፡ ስምከ፡
+                                 ይኩን፡ ይጻእ፡ ወይልሳሕ፡ ሕምዘ፡ ተመን፡ ሕምዘ፡ ከይሲ፡ ሕምዘ፡ አፍዖት፡ አንተ፡ ከይሲ፡ አይቴ፡
+                                 ነበርከ፡ ዘነሰከ፡ ወዘአንሰከ፡ ኅድግ፡ ይቤሉከ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ለገብርከ፡ እገሌ።
+                                 ዕፀ፡ ሰመግ፡ ዘዌ፡ የጸጅማጸባ፡ አ<add place="above">ም</add>ፋር፡ ቕፍርግ፡ <gap
+                                    reason="illegible" unit="chars" quantity="1"/><cb n="b"/>
+                                 እምስረዊሙ፡ ለእሉ፡ ዕፀዋት፡ አጥዕሞ፡ ለዘ<add place="above">ተ</add>ነስከ። ጸሐፊሁ፡
+                                 ገብረ፡ ሥላሴ፡ በመንግሥተ፡ ኢዮአስ፡ ንጉሥ፡ </q>
                            </item>
 
-                           <item xml:id="a3">
-                              <desc type="Comment">There are many glosses, interpolations,
-                                 emendations and erasures by a later hand.</desc>
-                              <q xml:lang="gez"/>
+                           <item xml:id="e3">
+                              <desc>Aids for identifying weekday readings were written in the upper
+                                 margin on the incipit page of each section of <ref target="#p2_i1"
+                                 /> but were all cut off, presumably during the trimming process,
+                                 see: <locus target="#4ra #23rb #53va #77ra #106vb #131ra #148vb"/>. </desc>
                            </item>
-
-                           <item xml:id="e1">
-                              <locus target="#20v"/>
-                              <locus target="#93"/>
-                              <desc>The name of the owner, <persName ref="PRS11353Constantine"
-                                    role="owner"/>, is mentioned on <locus target="#20v"/> and
-                                    <locus target="#93"/>.</desc>
-                              <q xml:lang="gez"/>
+                           <item xml:id="e4">
+                              <desc> A few corrections written interlineally or over erasures were
+                                 made by secondary hands (e.g. <locus target="#16rb #26vb #73va"/>),
+                                 especially in section <ref target="#p2_i1.5"/> where also
+                                 punctuation marks were corrected (e.g. on <locus
+                                    target="#77rb #78ra"/>: from ፡ to ፤, or on <locus target="#77vb #80rb"
```

Comment:
I am sorry to be a bit pedant, but since our : and the Ethiopic ፡ are really similar, why don't we rephrase the sentence into something like that:  `(e.g. from ፡ to ፤ on <locus target="#77rb #78ra"/>, or from ፡ to ። on <locus target="#77vb #80rb"/>).`?
I think this would help our readers who haven't a good sight and might be perplexed by those adjacent colons.

Reply:
Thanky you. No worries. I use Ethiopic Unicode, and ther eit is very celar. But you are absolutely right. I changed as suggested.